src/app/api/v1/user/email/route.ts

```diff
@@ -108,6 +107,6 @@ export async function POST(req: NextRequest) {
     }
   } else {
     // Not Signed in, redirect to home
-    return NextResponse.redirect(AppConstants.SERVER_URL, 301);
+    return NextResponse.redirect(process.env.SERVER_URL ?? "/", 301);
```

Comment:
I'm curious why you don't use the `getEnvVarsOrThrow` function here? Seems like you could avoid having to do the ` ?? "/"`, is this fallback ever desirable?